examples/haskell/example.hs

```diff
@@ -0,0 +1,16 @@
+
+import OpenCog.AtomSpace.Api
+import OpenCog.AtomSpace.Types
+import Control.Monad.IO.Class
+import Data.Default
+
+main :: IO ()
+main = runOnNewAtomSpace program
+
+program :: AtomSpace ()
```

Comment:
Never mind, I clearly need to understand more what's going on. We can discuss that during our next Handgout.